pkgs/development/web/nodejs/nodejs.nix

```diff
@@ -1,67 +1,64 @@
 { stdenv, fetchurl, openssl, python2, zlib, libuv, v8, utillinux, http-parser
-, pkgconfig, runCommand, which, libtool
-, version
-, sha256 ? null
-, src ? fetchurl { url = "https://nodejs.org/download/release/v${version}/node-v${version}.tar.xz"; inherit sha256; }
-, preBuild ? ""
-, extraConfigFlags ? []
-, extraBuildInputs ? []
-, patches ? [],
- ...
+, pkgconfig, runCommand, which, libtool, fetchpatch
+, callPackage
+, darwin ? null
+, disableNpm ? false
```

Comment:
I believe `enableNpm`, `npmSupport`, or `withNpm` naming styles are more popular to avoid an extra layer of negatives. (I would go for `enableNpm` personally.)